keda/templates/02-crd-clustertriggerauthentications.yaml

```diff
@@ -4,6 +4,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.9.0
+    helm.sh/hook: crd-install
```

Comment:
This is not supported anymore with helm v3. This was the mechanism of helm v2

Reply:
Hey I just checked that in helm v3 docs, they encourage to use a dedicated `crds` repository instead. I am willing to do it, does keda need to maintain the helm v2 compatibility at all ?

Reply:
I'd not change anything as its perfectly fine like it is for the KEDA chart. You should just not make an example with a resource of CRD kind the chart itself deploys.

If crds are moved outside the `/templates` folder into `/crds` the chart itself cannot update the CRDs if needed. Therefore I encourage you to not move anything regarding the CRDs.

Reply:
@mkilchhofer agree, good point.